repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
e.g. if we insist on a well-known `<package-name>.framework-name` config value for packages where `framework == true`, then the package manager can store this info as necessary.